### PR TITLE
Create CriteriaAudioTest class

### PR DESCRIPTION
**Pull request by LimbertVargas**
Hi guys,
I send the test of CriteriaAudio class for review.
please send me your comments

### CODE REVIEW

File: src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    /**
+     * This Test returns expected "2" audio channel when enter actual audio channel in getAudioChannel.
+     */
+    @Test
+    public void getAudioChannel() {
+        assertEquals(2,criteriaAudio.getAudioChannel());
+    }
+}
```
Comment (TheYisusGithub) [inline src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java]: Please insert a slat of line.

Comment (JosueRodriguezGarcia) [inline src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java]: please add a new line

File: src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+/**
```
Comment (alszla) [inline src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java]: Relocate this comment block just before class definition.

File: src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java (hunk 1 above)
Comment (LimbertVargas) [reply src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java]: fixed

Comment (LimbertVargas) [reply src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java]: fixed

File: src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java (hunk 2 above)
Comment (LimbertVargas) [reply src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java]: fixed

File: src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+/**
```
Comment (alszla) [inline src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java:17]: Please insert an extra line after the import statement.

File: src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+ * This class implements the CriteriaAudioTest class File and the getter methods for testing CriteriaAudio class.
+ *
+ * @author Limbert Alvaro Vargas Laura.
+ * @version 1.0
+ */
+public class CriteriaAudioTest {
+
```
Comment (alszla) [inline src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java]: I guess it'd be better removing this blank line

File: src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java (hunk 3 above)
Comment (LimbertVargas) [reply src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java:17]: fixed

File: src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java (hunk 4 above)
Comment (LimbertVargas) [reply src/test/java/com/fundation/webservice/model/CriteriaAudioTest.java]: fixed

**Pull request merged**